test/pummel/test-heapdump-quic.js

```diff
@@ -69,62 +69,76 @@ server.on('session', common.mustCall((session) => {
       }
     ], { loose: true });
 
-    state.validateSnapshotNodes('Node / QuicServerSession', [
+    state.validateSnapshotNodes('Node / QuicSession', [
       {
         children: [
           { node_name: 'QuicServerSession', edge_name: 'wrapped' },
-          { node_name: 'Node / rx_secret', edge_name: 'rx_secret' },
-          { node_name: 'Node / tx_secret', edge_name: 'tx_secret' },
+          { node_name: 'Node / QuicCryptoContext', edge_name: 'crypto_context'},
```

Comment:
Nit: A space if needed before the closing bracket:
```console
76:79  error  A space is required before '}'  object-curly-spacing
```

Reply:
```suggestion
          { node_name: 'Node / QuicCryptoContext', edge_name: 'crypto_context' },
```